openfasoc/generators/ldo-gen/flow/design/sky130hvl/ldo/constraint.sdc

```diff
@@ -1,3 +1,10 @@
 # create_clock [get_ports CLK_REF] -period 4
+set clk_name  core_clock
+set clk_port_name clk
+set clk_period 1000
```

Comment:
The clock should be a variable part of the modeling. Speed will impact the power, latency etc..

Reply:
@lulu9312 is this being looked at?

Reply:
Yes. This was pushed in the latest commit